src/lib/server/models.ts

```diff
@@ -37,6 +38,14 @@ const combinedEndpoint = endpoint.transform((data) => {
 	}
 });
 
+let parsedModels;
+try {
+	parsedModels = JSON.parse(MODELS);
+} catch (e) {
+	throw error(500, {
+		message: "There's a formatting error in your MODELS .env file. Please check it (missing/extra commas, etc.).",
```

Comment:
would be nice to be able to surface the line or property around which the JSON parser fails, as it's quite hard to find what's broken otherwise

Reply:
Yes the only way is to use another kind of JSON.parse for this one. Maybe https://github.com/zaach/jsonlint

Reply:
Resolve with jsonlint